docs/dcp2_system_design.rst

```diff
@@ -1824,3 +1824,61 @@ The ``supplementary_file.provenance.submitter_id`` field is set to
 ``c9efbb15-c50c-5796-8d15-35e9e1219dc5``. This UUIDv5 was generated using the
 same namespace UUID as for `Contributor-generated matrices (CGMs)`_ and the
 name ``dcp1 matrix service``.
+
+
+
+
+Access controls
+===============
+
+The DCP/2 manages access to the (meta)data that it hosts. When a *principal*
+(user, network, process, or application) that is not part of the DCP/2
+requests to retrieve a *resource* (metadata entity or data file) from a
+component of the DCP/2, the following rules apply:
+
+1) If the TDR snapshot containing the requested resource is marked public,
+   the request will be satisfied, independently of whether the principal's
+   identity is evident from the request or not.
+
+2) Otherwise, if the request is for metadata in the ``project`` entity, and
+   the request is made to the Azul REST API, the request will be satisfied.
+
+3) Otherwise, if the request is for a `contributor-generated project-level
+   matrix data file <Contributor-generated matrices (CGMs)_>`_, and the request
+   is made to the Azul REST API, the request will be satisfied.
+
```

Comment:
What about DCP-generated matrices?

Reply:
I agree that both contributor-generated and DCP-analysis-generated project-level matrices should always be open access.

Reply:
Sorry, KT asks a question and T agrees with someone but I don't know who with.

Can I get a clear directive on this? I will work in whatever you decide.

Reply:
Sorry!  Yes, we should make DCP-generated matrices available too.  I should have phrased it better.

Reply:
I'll incorporate that.

Reply:
Fixed.